doc/fornax_setup.md

```diff
@@ -0,0 +1,245 @@
+# Fornax Core and Nodeagent Setting Up Detail
+
+## Overview
+This doc is the detail steps which setting up, configuration and install component for Fornax Serverless Machine.
+
+## Machine Prepare
+If you have a brand new machine, your should install following component to start the machine. (Following is detail steps for AWS EC2 virtual machine).
+
+```script
+sudo apt-get update
+sudo apt install build-essential
+sudo snap install curl  # version 7.81.0
+sudo apt-get install vim
```

Comment:
why do we need vim and build-essential, it's not for developer

Reply:
If new virtual machine created by AWS (brand new VM), we need install this build-essential and vim. sometime default vim is old version with VM